sdk/nodejs/api/client.gen.ts

```diff
@@ -219,26 +271,41 @@ export type DirectoryPipelineOpts = {
 export type DirectoryWithDirectoryOpts = {
   /**
    * Exclude artifacts that match the given pattern.
-   * (e.g. ["node_modules/", ".git*"]).
+   * @example ["node_modules/", ".git*"]
    */
   exclude?: string[]
 
   /**
    * Include only artifacts that match the given pattern.
-   * (e.g. ["app/", "package.*"]).
+   * @example ["app/", "package.*"]
    */
   include?: string[]
 }
 
 export type DirectoryWithFileOpts = {
+  /**
+   * Permission given to the copied file.
+   * Default to 0644.
```

Comment:
Is it possible to use `@default` instead of `Default`. If not it's fine.

Reply:
Possible but I would like to keep it for a follow-up PR if possible!

Reply:
Fixed in https://github.com/dagger/dagger/pull/4367/commits/2a0ebdce4d802f8666d5963ca8963641169fd915

Reply:
I don't see any updates, Default is still the same. But honestly like I've said it's not a big deal and like you've said it can be addressed in a following PR.